coordinator.Dockerfile

```diff
@@ -8,7 +8,7 @@ COPY ./ ./
 
 RUN cd ./cmd/coordinator && CGO_ENABLED=0 go build -ldflags "-s -w -extldflags '-static'" -o coordinator
 
-FROM --platform=linux/amd64 scratch AS runtime
+FROM --platform=linux/amd64 golang:1.22-alpine AS runtime
 
 COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
can delete this, and use 
RUN apk add --no-cache ca-certificates

like bootnode.Dockerfile
same as blow